pom.xml

```diff
@@ -2611,9 +2611,9 @@
             </args>
             <jvmArgs>
               <jvmArg>-Xss128m</jvmArg>
-              <jvmArg>-Xms4g</jvmArg>
-              <jvmArg>-Xmx4g</jvmArg>
-              <jvmArg>-XX:MaxMetaspaceSize=2g</jvmArg>
+              <jvmArg>-Xms6g</jvmArg>
+              <jvmArg>-Xmx6g</jvmArg>
+              <jvmArg>-XX:MaxMetaspaceSize=512m</jvmArg>
```

Comment:
FYI, Under 1G, hive job failed with `OOM: Metaspace`. You can see the error message in the original PR.

Reply:
I once reduced it to 128m and see `OOM: Metaspace`. Currently, 512m is still running. If it fails, probably increasing it back.

Reply:
It will run with errors. Last time, Jenkins runs forever and was killed by timeout.

Reply:
Yea, 1g seems not enough too.. Trying to add a bit more.